sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2364,6 +2364,21 @@ public void deleteCase(String caseUuid) throws ExternalSurveillanceToolException
 		caseService.delete(caze);
 	}
 
+	public List<String> deleteCases(List<String> caseUuids) {
```

Comment:
Return void instead of a list of Strings and don't execute `isDeleted` for every case because that means one additional database call and query each time; if something goes wrong during the deletion, there will likely be an error anyway; also, please add the user right check from the `deleteCase` method at the start of this method as well

Reply:
The guys from the frontend requested a return. What about if I return similar as in case of push instead of PushResult, and Im not checking I return error only if that is an error?